src/components/SegmentedProgressBar/SegmentedProgressBar.jsx

```diff
@@ -32,10 +38,21 @@ export default function SegmentedProgressBar({ current, total, label }) {
 }
 
 SegmentedProgressBar.propTypes = {
+  /**
+   * The `aria-valuetext` to use for the component
+   */
+  ariaValueText: PropTypes.isRequired,
```

Comment:
According to the ticket, I'm only adding `ariaValueText` as a prop to the `<SegmentedProgressBar>`